### PR TITLE
chore(AGENTS.md): prefer `whenStable` in unit tests

### PR DESCRIPTION
**Pull request by spike-rabbit**
Angular recommends this approach.

---

[Documentation](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/).
[Examples](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/element-examples/).
[Dashboards Demo](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/dashboards-demo/).
[Playwright report](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/playwright-report/).

**Coverage Reports:**

Code Coverage

- [charts-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/charts-ng/)
- [dashboards-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/dashboards-ng/)
- [element-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/element-ng/)
- [element-translate-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/element-translate-ng/)
- [maps-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/maps-ng/)
- [native-charts-ng](https://d33c9dcnqinn2a.cloudfront.net/pr-1954/pages/coverage/native-charts-ng/)

### CODE REVIEW

File: AGENTS.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ You are an expert in TypeScript, Angular, and scalable web application developme
 - Keep each test focused on a single behavior with clear arrange / act / assert phases
 - Cover happy paths, boundary conditions, and regression-prone branches. Avoid broad snapshot-style assertions that do not explain the intended behavior
 
+## Async Test Stabilization
+
+- Prefer `await fixture.whenStable()` over repeated `fixture.detectChanges()` calls after interactions or async state changes
+- `whenStable()` waits for pending microtasks, timers, and zone activity to settle, producing more reliable tests than manually pumping change detection
+- Use `fixture.detectChanges()` deliberately for the initial render or when the change detection boundary itself is under test
+- Do NOT chain multiple `detectChanges()` calls hoping to flush async work — use `whenStable()` instead
```
[the remark this answers is not in the document]
Comment (spike-rabbit) [reply AGENTS.md:44]: those rules don't apply here. It is technical for AI

**Review by spliffone — APPROVED** [review]

**Review by kfenner — APPROVED** [review]
LGTM 👍
